oss_src/unity/python/sframe/cython/cy_sarray.pxd

```diff
@@ -175,6 +176,8 @@ cdef class UnitySArrayProxy:
 
     cpdef sample(self, float percent, int seed)
 
+    cpdef hash(self, int seed)
```

Comment:
ditto. or just do a python object, then call hash() on that.  then it's a 64bit int.